applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/Kconfig

```diff
@@ -217,6 +217,21 @@ config BT_AUDIO_SCAN_DELEGATOR
 	  search for a predefined broadcast source. Instead, it will wait for a
 	  broadcast assistant to connect and control.
 
+config BT_SET_IDENTITY_RESOLVING_KEY_DEFAULT
+	string
+	default "NRF5340_BIS_DEMO"
+	help
+	  Default string to configure the Set Identify Resolving Key (SIRK), must
+	  be changed before production uniquely for each coordinated set.
+
+config BT_SET_IDENTITY_RESOLVING_KEY
+	string "String used to configure the SIRK"
+	default BT_SET_IDENTITY_RESOLVING_KEY_DEFAULT
```

Comment:
Do we need both of these, or could we put in the default key on this line?

Reply:
I reused this part from unicast server, but I changed the default key to avoid phone tried to connect to both demos.
Will create a task in Jira which split CSIS as an independent module which can be share between CIS/BIS demo.